includes/lib/transifex-live-integration-wp-services.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * Language rewrites
```

Comment:
`Language rewrites` doesn't seem accurate for this class. Maybe add a couple of lines of what should be added here?

Reply:
You're right it's not accurate, will remove. No need for extra description since we follow the convention 1 class per file and the class contains docstrings.